src/site/layouts/page.drupal.liquid

```diff
@@ -15,7 +15,9 @@
         {{ fieldIntroText }}
       </div>
       {% for block in fieldContentBlock %}
-        {{ block.entity.entityRendered }}
+        {% assign bundleComponent = "src/site/bundles/" | append: block.entity.entityBundle %}
+        {% assign bundleComponentWithExtension = bundleComponent | append: ".drupal.liquid" %}
+        {% include {{ bundleComponentWithExtension }} with block = block %}
```

Comment:
This is the dynamic import for new bundle templates. I'm sure there's a more elegant way to do this, but either way it does what it needs to do.

Reply:
What happens if our template doesn't exist yet for a bundle? I can see that happening as they add new things in Drupal. Does the build process break?

Reply:
It doesn't break - it just skips that block as if there wasn't an include at all. I think we could add a filter to validate that the bundle template exists though if we do want to break it or log a warning?

Reply:
I think a warning would be helpful, but we can do that later.